woocommerce/woocommerce-admin
# Marketing inbox note

Fixes #4029

Adds a note to introduce merchants to the new Marketing Tab/Dashboard/Hub.

**Update** - ([see comment below](https://github.com/woocommerce/woocommerce-admin/pull/4030#issuecomment-606925760)) after some discussion (pbIy4N-3C-p2#comment-127) we've decided to remove two existing inbox notes and replace with this one note directing merchants to the marketing hub.

---

I have not currently put any restrictions on displaying the new note because my understanding is that we want to cover everyone e.g. both
- users that don’t finish the task list (hide it / click to do it later)
- users that complete the task list

I guess that is everyone? If I'm missing some logic needed let me know.

Text currently displayed in the note is
- **Title:** WooCommerce marketing hub
- **Content:** Grow your customer base and increase your sales with marketing tools built for WooCommerce.
- **Action:** Open marketing hub 

Notes
- I went with "marketing hub" to match other community information but am happy to change any of the wording @garymurray @LevinMedia
- Similarly I went with using "open" as the action seemed better than something like "learn more"
- Avoided using something like "New xyz" for the title because it might not always be new. Happy to tweak.

Here is a screenshot with a few of the variations I tried

<img width="513" alt="Screen Shot 2020-03-31 at 4 14 25 pm" src="https://user-images.githubusercontent.com/355014/77992090-e393d500-736c-11ea-8ca6-1342817ac0fd.png">

### Detailed test instructions:

* Checkout this branch
* Trigger the wc_admin_daily hook scheduled in WP Cron
* Go to a WooCommerce page
* Click the Inbox icon in the Navbar
* Observe the new note
* Click the action button to be directed to the new Marketing tab
